# -*- coding: utf-8 -*-
# Generated by Django 1.9.8 on 2017-01-18 15:14
from __future__ import unicode_literals

from django.conf import settings
import django.contrib.auth.models
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        ('auth', '0007_alter_validators_add_error_messages'),
    ]

    operations = [
        migrations.CreateModel(
            name='Empresa_a',
            fields=[
                ('user_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
                ('sexo', models.CharField(choices=[('M', 'Masculino'), ('F', 'Femenino'), ('O', 'Otro')], max_length=1)),
                ('empresa', models.CharField(max_length=50)),
                ('domicilio', models.CharField(max_length=70)),
                ('nit', models.IntegerField()),
                ('Doc', models.IntegerField()),
                ('telefono', models.CharField(max_length=12)),
                ('firma', models.ImageField(upload_to='firma')),
                ('imgperfil', models.ImageField(null=True, upload_to='perfil')),
            ],
            options={
                'verbose_name_plural': 'users',
                'abstract': False,
                'verbose_name': 'user',
            },
            bases=('auth.user',),
            managers=[
                ('objects', django.contrib.auth.models.UserManager()),
            ],
        ),
    ]
